### DISCUSSION TITLE
GitHub Pages repo size limit - compressed or uncompressed?

### DISCUSSION THREAD

**Discussion by briankung-bpn** [community/community#50594] [Other Feature Feedback, Questions, & Ideas]
### GitHub Pages repo size limit

Does the GitHub Pages repo size limit refer to its compressed or uncompressed size?

### Body

I have a GHA that will upload some Rust documentation to GitHub Pages on merge to a branch. In the Actions logs, I see this message:

```
The raw size of all the files that were specified for upload is 940933120 bytes
The size of all the files that were uploaded is 75678220 bytes. This takes into account any gzip compression used to reduce the upload size, time and storage
```

On [this page](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits) it says:

> Published GitHub Pages sites may be no larger than 1 GB.

Does that refer to the compressed size or the uncompressed size? And does 1 GB refer to 1000MiB or 1GiB (1024 MiB)?

**Comment by yoannchaudet** [comment]
👋 I know there can be confusion between GB and GiB but here we mean 1 billion bytes. We use this limit to protect the service against abuse and the semantic of it is uncompressed data. ~With your site approaching 10GB you are way over our recommended limit and I strongly recommend you find ways to reduce the size of your site.~

**Reply by briankung-bpn** [reply]
We are at `0.94 gigabytes` if I calculated correctly: `940_933_120 bytes`. Would you be able to answer if the limit refers to compressed or uncompressed? We will reduce the generated static files if we exceed the limit.
<img width="787" alt="Screenshot 2023-03-21 at 3 55 10 PM" src="https://user-images.githubusercontent.com/119887860/226738586-8b84f851-be68-4ddc-b8a0-bfe2b5b29174.png">

**Reply by yoannchaudet** [reply]
Oups that's me not knowing how to count properly. You are all good and could go a little over, it would be fine as well. Like I said the limit is used to protect the service, and is enforce on a need basis.

**Reply by briankung-bpn** [reply]
Thank you!
